ts/Data/DataPool.ts

```diff
@@ -346,6 +363,11 @@ class DataPool implements DataEvent.Emitter {
             }
         }
 
+        if (instances[options.id]) {
+            instances[options.id].stopPolling();
```

Comment:
Why do we stop pooling here? If declared, is it initialized once again? 
Maybe it's worth checking if the `polling` is even enabled first?

Reply:
Yes, here the connector gets replaced, and when it gets disposed from the data pool, then there is no point in polling anymore. Also `stopPolling` covers all conditions, no need for an extra check.